wisdem/inputs/modeling_schema.yaml

```diff
@@ -356,6 +361,9 @@ properties:
                         description: Set of Frame3DD options used for floating tower analysis
                         default: {}
                         properties:
+                            flag:
+                                <<: *flag
```

Comment:
Your yaml skills have always been sharp.  What is this notation?  Does this mean use the prior definition of flag but just update the default?  That would be so helpful in the schema in so many other places!

Reply:
The `<<` is a [merge key](https://yaml.org/type/merge.html), and that's exactly how I'm using it here: all the flag info is merged, but we want the default to instead be `false`.